src/main/resources/liquibase/crop_changelog/20_2_0.xml

```diff
@@ -28,4 +28,41 @@
 		</sql>
 	</changeSet>
 
+	<changeSet author="cheryleigh" id="v20.2.0-3">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="external_reference_atributs"/>
```

Comment:
Noting here that I've created [IBP-5516](https://ibplatform.atlassian.net/browse/IBP-5516) to created audit trail for this new table: `external_reference_atributs`. Note that we implement audit trail for all germplasm related tables